aospy/test/test_calc_basic.py

```diff
@@ -37,6 +39,10 @@ def test_annual_mean(self):
         calc.compute()
         assert isfile(calc.path_out['av'])
         assert isfile(calc.path_tar_out)
+        data = xr.open_mfdataset(calc.path_out['av'], decode_times=False)
+        for name, da in data.data_vars.items():
+            assert 'units' in da.attrs
```

Comment:
Instead of testing just that the attribute was added, I recommend testing that the value of the attribute is what you expect.  This will test both that the attribute is in the dictionary, and make sure the proper value was encoded simultaneously.

I think you pick off the expected values of the units and description attributes from the `Var` object associated with the test:
```python
expected_units = calc.var.units
expected_description = calc.var.description
```